_posts/2024-08-24-GSoC2024-dingo.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: single
+title: "Enhancing dingo: from metabolic models reduction to pathways identification"
+date: 2024-08-24
+author: Sotiris Touliopoulos
+author_profile: true
+read_time: true
+comments: true
+share: true
+related: true
+hidden: false
+---
+
+
+# Enhancing dingo: from metabolic models reduction to pathways prediction
+
+> #### A contribution to the Google Summer of Code 2024 program
+
+
+## Overview
+
+#### A summary of the implemented methods, integrated into dingo:
+
+- Preprocess for the reduction of metabolic models.
+- Inference of pairwise correlated reactions.
+- Visualization of a steady-states correlation matrix.
+- Clustering of a steady-states correlation matrix.
+- Construction of a weighted graph of the model's reactions with the correlation coefficients as weights.
+
+
+## Preprocess
+
+- Large metabolic models contain numerous reactions and metabolites.
+- Sampling the flux space of such models requires significant computational time due to the high dimensionsionality. 
+- Model preprocessing can mitigate this issue by removing certain reactions, thus reducing the dimensional space.
+
+A `PreProcess` class was implemented. After initializing an object from this class, 
+users can call the `reduce` function to remove 3 types of reactions:
+
+- Blocked reactions: cannot carry a flux in any condition.
+- Zero-flux reactions: cannot carry a flux while maintaining at least 90% of the maximum growth rate.
+- Metabolically less-efficient reactions: require a reduction in growth rate if used.
+
+Users can choose to remove an additional set of reactions, by setting the `extend` parameter 
+of the `reduce` function to `True`. These reactions do not affect the value of the objective function when removed.
+
+Reduction with the `PreProcess` class has been tested with various models, some of which are included in dingo’s publication article too. Figures below show the number of remained reactions, after calling the `reduce` function.
```

Comment:
I would remove the dingo's publication part and mention that they are BiGG models.